### DISCUSSION TITLE
Why does input with border now only show the grey border after it is focused?

### DISCUSSION THREAD

**Discussion by PizzaConsole** [saadeghi/daisyui#2017] [Q&A]
In version 2.52.0 always had a grey outline always and highlighted while focused. Now in v3.1.1 its different.

![image](https://github.com/saadeghi/daisyui/assets/60050783/bd91cc21-016c-44e1-822d-a2bd11a5eaf2)

![image](https://github.com/saadeghi/daisyui/assets/60050783/e720fcdc-70f4-4dac-8e71-9d82366c47b8)

**Comment by akkie** [comment]
Focus was always there. After the release of version 3.1.0, the borders are not really visible anymore. Version 3.0.29 does not have this issue. So I guess it's an issue. Please vote: https://github.com/saadeghi/daisyui/issues/2055
